entc/gen/type.go

```diff
@@ -172,7 +172,10 @@ func NewType(c *Config, schema *load.Schema) (*Type, error) {
 	typ := &Type{
 		Config: c,
 		ID: &Field{
-			Name:      "id",
+			Name: "id",
```

Comment:
You will need to an extra test case for this scenario, one where the default id field is used

Reply:
There is already one present in `type_test.go`

https://github.com/facebook/ent/pull/991/files#diff-5c305024b752f5d6230be7644ec718d9a78debb409e9bd56ebeb69dff6bbb34eR224-R226

Do you mean an extra integration test?

Reply:
Yup. Like the card one mentioned below

Reply:
@cliedeman added in 083aeec and 19187e5